src/Classes/OystPrice.php

```diff
@@ -81,11 +81,11 @@ public function setCurrency($currency)
      */
     public function toArray()
     {
-        $tax = array(
+        $price = array(
```

Comment:
```$amount``` is a better wording, to match the API which is probably like : https://www.adyen.com/apidocs/?example=Cancel%20Or%20Refund%201